fakebook/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  baseUrl: 'someUrl' //replace with backend production url
+  baseUrl: 'http://localhost:4200/' //replace with backend production url
```

Comment:
that's not the production url, and the structure of this file must not differ from the structure of environment.ts. that's fundamental.

Reply:
Fixed with actual url